src/frontend/render/settings.rs

```diff
@@ -15,7 +15,7 @@ use trippy::tracing::PortDirection;
 pub fn render(f: &mut Frame<'_>, app: &mut TuiApp) {
     let all_settings = format_all_settings(app);
     let (name, info, items) = &all_settings[app.settings_tab_selected];
-    let area = util::centered_rect(60, 60, f.size());
+    let area = util::centered_rect(75, 60, f.size());
```

Comment:
Why do we need to do this?

Reply:
Ahh a leftover from the previous output of ['H', 'O', 'L', 'S', 'R', 'A', 'V', 'B', 'W', 'D', 'T'] which got truncated in the setting pop-up.  The suggestion of displaying HOLSRAVBWDT should fit in the 60,60 rect. 
Plan is to revert to 60,60 rect.